exercise46.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+FILENAME = 'words.txt'
+string_of_words = ''
+
```

Comment:
@paulghaddad can use list comprehenion when opening and reading a file?

Reply:
No, you'll always want to use a look when dealing with files...that's what you normally see.